doc/nb/keck_lris_blue_long_400_3400_d560.cfg

```diff
@@ -1,37 +1,27 @@
-[run]
-    ncpus = 1
-    verbosity = 2
-    overwrite = True
-    sortdir = lris_blue_long_400_3400_d560
-
 [rdx]
-    spectrograph = KECK_LRISb
+    spectrograph = keck_lris_blue
     pipeline = ARMS
+
+[calibrations]
     [[flatfield]]
         frame = /Users/westfall/Work/packages/PYPIT-development-suite/CALIBS/PYPIT_LRISb_pixflat_B400_2x2_15apr2015.fits.gz
```

Comment:
Is there a way to specify this more generally for each user?

Reply:
Yes, ConfigObj has a way that you can define a path and then use it (like when setting an environmental variable) and use it in other places in the file.  This isn't as powerful as in the normal ConfigParser class, though.  We can talk more about this on a telecon, and I just added #402.